server/services/expectedArrivalsService.ts

```diff
@@ -77,23 +83,27 @@ export default class ExpectedArrivalsService {
   public async getArrivalsForToday(
     agencyId: string,
     now = () => moment()
-  ): Promise<Map<LocationType, WithBodyScanInfo<Arrival>[]>> {
+  ): Promise<Map<LocationType, DecoratedArrival[]>> {
     const [expectedArrivals, transfers] = await Promise.all([
       this.getExpectedArrivals(agencyId, now()),
       this.getTransfers(agencyId),
     ])
-    const allArrivals = await this.bodyScanDecorator.decorate([...expectedArrivals, ...transfers])
-    return groupBy(allArrivals, (arrival: WithBodyScanInfo<Arrival>) => arrival.fromLocationType)
+    const allArrivals = [...expectedArrivals, ...transfers]
+    const withBodyScan = await this.bodyScanDecorator.decorate(allArrivals)
+    const withBodyScanAndMatchType = this.matchTypeDecorator.decorate(withBodyScan)
+
+    return groupBy(withBodyScanAndMatchType, (arrival: DecoratedArrival) => arrival.fromLocationType)
```

Comment:
This is required so we can determine on the list view what kind of match each prisoner is - we'll use this to determine whether to show the photo or not